freqtrade/exchange/exchange.py

```diff
@@ -1508,6 +1508,35 @@ def get_historic_trades(self, pair: str,
             self._async_get_trade_history(pair=pair, since=since,
                                           until=until, from_id=from_id))
 
+    def get_funding_fees(self, pair: str, since: Union[datetime, int]) -> float:
```

Comment:
You will want the "retrier" decorator - otherwise raising DDOS errors will NOT retry the call - but simply "dump" that out into whatever called it.